### PR TITLE
EDM-621: Fix microshift registration docs

### PR DESCRIPTION
**Pull request by avishayt**
1. ca.crt in repository
2. verificationSuffix in repository (EDM-648)
3. pull-secret in fleet spec

### CODE REVIEW

File: docs/user/registering-microshift-devices-acm.md [hunk 1]
```diff
@@ -57,18 +58,20 @@ spec:
       os:
         image: quay.io/myorg/device-image-with-microshift:v1
       config:
-      - configType: HttpConfigProviderSpec
+      - name: acm-crd
         httpRef:
           filePath: /var/local/crd.yaml
```
Comment (fzdarsky) [inline docs/user/registering-microshift-devices-acm.md:63]: Could we write that to a subdirectory below `/var/local` instead, so we keep this system directory clean?

File: docs/user/registering-microshift-devices-acm.md [hunk 2]
```diff
@@ -92,29 +95,36 @@ As described in the user documentation, a fleet specification is composed of var
 1. **Configuration**: The configuration section uses the HTTP configuration provider to fetch information from an endpoint. The repository `acm-test` contains the registration URL for ACM's agent registration:
 
     ```console
-    - configType: HttpConfigProviderSpec
+    - name: acm-crdv
```
Comment (fzdarsky) [inline docs/user/registering-microshift-devices-acm.md:98]:
```suggestion
    - name: acm-crd
```

File: docs/user/registering-microshift-devices-acm.md (hunk 1 above)
Comment (avishayt) [reply docs/user/registering-microshift-devices-acm.md]: Changed to /var/local/acm-import/

File: docs/user/registering-microshift-devices-acm.md [hunk 3]
```diff
@@ -57,18 +58,20 @@ spec:
       os:
         image: quay.io/myorg/device-image-with-microshift:v1
       config:
-      - configType: HttpConfigProviderSpec
+      - name: acm-crd
         httpRef:
           filePath: /var/local/crd.yaml
           repository: acm-test
```
[the remark this answers is not in the document]
Comment (avishayt) [reply docs/user/registering-microshift-devices-acm.md]: Changed the acm-test repo to acm-registration, changed the selector to fleet-acm

File: docs/user/registering-microshift-devices-acm.md (hunk 2 above)
Comment (avishayt) [reply docs/user/registering-microshift-devices-acm.md]: Fixed typo.
Yes, it's expected that it's owned by root.

**Review by avishayt — COMMENTED** [review]
Fixed comments

**Review by celdrake — COMMENTED** [review]
LGTM

File: docs/user/registering-microshift-devices-acm.md [hunk 4]
```diff
@@ -24,16 +25,16 @@ apiVersion: v1alpha1
 kind: Repository
 metadata:
   labels: {}
-  name: acm-test
+  name: acm-registration
 spec:
   httpConfig:
     token: $token
+    ca.crt: <base64-encoded ca.crt>
```
Comment (fzdarsky) [inline docs/user/registering-microshift-devices-acm.md:32]: Can we describe where the user would get this from?

**Review by fzdarsky — APPROVED** [review]

**Pull request merged**